config/defaults.json

```diff
@@ -0,0 +1,135 @@
+{
+  "cli_wallet_connection": "",
+  "db": {
```

Comment:
I would love to switch to Postgres in prod eventually, it is vastly preferable to mysql.